index.html

```diff
@@ -34,6 +39,19 @@
 				</div>
 			</nav>
 		</div>
+		<div class="mobile-nav" id="myLinks">
```

Comment:
id="myLinks" doesn't mean anything to me, if you decide to keep it, give it a good name.